wagtail/tests/test_page_model.py

```diff
@@ -2044,6 +2044,31 @@ def test_copy_unsaved_page(self):
         with self.assertRaises(RuntimeError):
             new_page.copy()
 
+    def test_copy_page_with_unique_uuids_in_orderables(self):
```

Comment:
I don't think this test does what it needs to.

I've dropped the changes in `wagtail/actions/copy_page.py` and the test is still green.

Reply:
🤔  Interesting! I drop the changes and the test fails for me. There are two areas the new translation key is generated if you run just the one test I wrote it will only affect the first implementation. If you run tests on the full test_page_models.py file it'll hit both implementations.

Reply:
I've pushed only the test to a branch - https://github.com/wagtail/wagtail/compare/translation-key-fix-test-only. The expectation is that the CI should fail, but
https://github.com/wagtail/wagtail/actions/runs/2097304339 is all green

Reply:
Hey @zerolab I did some tinkering locally, then removed the code I added in this PR, push it and waited for tests to pass - and they failed (which is what we want). The last commit `c9862be` re-added the code and the tests seem to be passing now (as expected).

Reply:
is your branch up to date with `main`? if so, will have a look tomorrow

Reply:
@zerolab Yep, it is up to date with `main`. 🎉